src/config/client_config.cc

```diff
@@ -4,7 +4,7 @@
 
     client_config.cc - this file is part of Gerbera.
 
-    Copyright (C) 2020 Gerbera Contributors
+    Copyright (C) 2021 Gerbera Contributors
```

Comment:
Wouldn't it be better to have 2020-2021?

Reply:
Yes, this was just me lazily find and replacing!

Reply:
I managed to break bitmap images in this way (replace in all files)